### ISSUE TITLE
Custom "dont-stack-until" for a entity

### ISSUE THREAD

**Issue by dudu65** [Nathat23/StackMob-5#97]
Hello,
The "dont-stack-until:" doesn't not work for custom entity:

custom:
    PIG:
        stack-max: 25
        dont-stack-until: 6
        tag:
            format: '%size%x Piggy'

Thanks.

**Comment by Nathat23** [comment]
you are not using Stackmob 5?

**Comment by dudu65** [comment]
Yes, just i want a global dont-stack-until to 5 for example and a dont-stack-until to 2 only for the pig you understand ?
